src/views/catalog/CreateVMHorizontalNav/CreateVMHorizontalNav.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+
+import CreateFromInstanceType from '@catalog/CreateFromInstanceTypes/CreateFromInstanceType';
+import TemplatesCatalog from '@catalog/templatescatalog/TemplatesCatalog';
+import { ALL_NAMESPACES } from '@kubevirt-utils/hooks/constants';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Stack, StackItem, Tab, Tabs, Title } from '@patternfly/react-core';
+import { CatalogIcon, ImageIcon } from '@patternfly/react-icons';
+
+import CreateVMTabTitle from './components/CreateVMTabTitle/CreateVMTabTitle';
+import { CREATE_VM_TAB } from './constants';
+
+import './CreateVMHorizontalNav.scss';
+
+const CreateVMHorizontalNav: FC<RouteComponentProps<{ ns: string }>> = ({
+  history,
+  location,
+  match,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [currentTab, setCurrentTab] = useState<CREATE_VM_TAB>(
+    location.pathname.includes(CREATE_VM_TAB.INSTANCE_TYPES)
+      ? CREATE_VM_TAB.INSTANCE_TYPES
+      : CREATE_VM_TAB.CATALOG,
+  );
+
+  const handleTabClick = (
+    event: React.MouseEvent<any> | React.KeyboardEvent | MouseEvent,
+    tabIndex: CREATE_VM_TAB,
+  ) => {
+    setCurrentTab(tabIndex);
+    history.push(
+      `/k8s/${
+        match?.params?.ns ? `ns/${match?.params?.ns}` : ALL_NAMESPACES
+      }/templatescatalog${tabIndex}`,
+    );
+  };
+
+  return (
+    <div>
+      <div className="pf-c-page__main-breadcrumb">
+        <Stack hasGutter>
+          <StackItem className="co-m-pane__heading">
+            <Title headingLevel="h1">{t('Create new VirtualMachine')}</Title>
+          </StackItem>
+          <StackItem>{t('Select an option to create a VirtualMachine from.')}</StackItem>
```

Comment:
We can ask Yifat about this if you want, but the correct English would be "Select an option from which to create a VirtualMachine."